src/reporter.js

```diff
@@ -31,7 +25,7 @@ export default function (message) {
         }
       }
     }, function() {});
-  }
 
-  console.log("AccessLint warnings: ", violations);
+    console.warn(violations);
```

Comment:
@geoffharcourt I think that could work. More important is deduping any identical errors. I think once we dedupe there will be much less noise.